src/views/portals/YetiView.vue

```diff
@@ -595,6 +778,59 @@
         result += '&username=' + this.$user.forumUsername;
 
         return result;
+      },
+
+      onMapMoveEnd() {
+        this.setVisibleMountains();
+      },
+
+      setVisibleMountains() {
+        const mapExtent = this.$refs.map.getExtent('EPSG:4326');
+        // clone this.mountains first, with no reference
+        this.visibleMountains = Object.assign({}, this.mountains);
```

Comment:
one could use object destructuring insteadof `Object.assign`

Reply:
Yes. But codebase is using `Object.assign` where needed (16 matches), not object destructuring (0 match).